addon/validators/base.js

```diff
@@ -85,25 +85,24 @@ export default Ember.Object.extend({
    * @param  {Object} context     : Context for string replacement
    * @return {String}             : The generated message
    */
-  createErrorMessage(type, options, value, context) {
+  createErrorMessage(type, options, value, context = {}) {
     options = options || {};
```

Comment:
Actually, doing that would change the method signature which would have to be propagated through each validator and it would also cause a change in the message function feature.